demo/DimImagesTransformer.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+<html lang=en>
+
+  <head>
+    <meta charset=utf-8>
+    <title>LazyLoadTransformer</title>
+    <link href=../build/wikimedia-page-library-transform.css rel=stylesheet>
+    <style>
+      body, img, input { margin: 14px; }
+    </style>
+  </head>
+
+  <body>
+
+    <input type='checkbox' onclick='
+      pagelib.DimImagesTransform.dimImages(
+        window, 
+        !pagelib.DimImagesTransform.isDimmingEnabled(window)
+      )
+    '> Dim images
+    <br>
+    
+    <script src=../build/wikimedia-page-library-transform.js></script>
+    <script src=./DemoImages.js></script>
+    <script>
+      /* global DemoImageData */
+      const body = document.getElementsByTagName('body')[0]
```

Comment:
In the production code you used querySelector() and here you used getElementsByTagName(). Any reason to prefer one over the other? I was operating under the idea that the result of getElementsByTagName() was live (changing), so I just kind of shied away from it.